src/main/java/uk/gov/hmcts/reform/hmc/api/controllers/HearingsController.java

```diff
@@ -325,6 +328,44 @@ public ResponseEntity<Object> getFutureHearings(
         }
     }
 
+    /**
+     * End point to create hearing details for a case.
+     *
+     * @return success response, and initiates async process to create hearings
+     *     passed.
+     * @header authorization, user authorization token.
+     * @header serviceAuthorization, S2S authorization token.
+     * @RequestBody caseDetails to take all the hearings belongs to case.
+     */
+    @PostMapping(path = "/automated-hearing")
+    @ApiOperation("create automated hearing")
+    @ApiResponses(
+            value = {
+                @ApiResponse(code = 200, message = "Initiated Async Create hearings successfully"),
```

Comment:
Is it Sync or Async?

Reply:
its a sync ... Thank you @shashisk97 addressed it